src/event/event.service.ts

```diff
@@ -211,11 +212,12 @@ export class EventService {
 
     if (search) {
       eventQuery.andWhere(
-        `(event.name LIKE :search OR 
-          event.description LIKE :search OR 
-          event.location LIKE :search OR 
-          CAST(event.lat AS TEXT) LIKE :search OR 
-          CAST(event.lon AS TEXT) LIKE :search)`,
+        `(event.name LIKE :search OR
+          event.slug LIKE :search OR 
+          event.description LIKE :search OR
```

Comment:
due to `description` field volume and missing index on that this query gonna be extremely long

Reply:
Any ideas here @codextech ?  Do we need description here?  Do we add an index on description?  Do we ignore it and create an issue to come back and resolve ?